content/blogposts/2022/release-4.1.md

```diff
@@ -0,0 +1,65 @@
+---
+title: "Sourcegraph 4.1 release"
+publishDate: 2022-10-21T10:00-07:00
+description: "Sourcegraph 4.1 introduces new features for server-side batch changes and improvements to the code navigation reference panel."
+tags: [blog, release]
+slug: "release/4.1"
+published: true
+heroImage: https://storage.googleapis.com/sourcegraph-assets/blog/release-post/4.1/sourcegraph-4-1.png
+socialImage: https://storage.googleapis.com/sourcegraph-assets/blog/release-post/4.1/sourcegraph-4-1.png
+changelogItems:
+  - description: Added repository sync counters to the code host details page to give visibility into external service sync progress.
+    url: https://github.com/sourcegraph/sourcegraph/pull/43039
+    category: Repositories
+  - description: "Added a new button in the repository settings, under \"Mirroring\", to delete a repository from disk and reclone it. This prevents the need to manually delete failed repositories from the Git server."
+    url: https://github.com/sourcegraph/sourcegraph/pull/42177
+    category: Repositories
+  - description: "GraphQL request logs are now compliant with the audit logging format. The old GraphQl logging based on `LOG_ALL_GRAPHQL_REQUESTS` env var is now deprecated and scheduled for removal."
+    url: https://github.com/sourcegraph/sourcegraph/pull/42550
+    category: Admin
+  - description: "Git server access logs are now compliant with the audit logging format. This introduces a breaking change: The 'actor' field is now nested under the 'audit' field."
+    url: https://github.com/sourcegraph/sourcegraph/pull/41865
+    category: Admin
+  - description: Security events are now included in the audit log by default.
```

Comment:
Any benefit/so what statement? Would it make sense to combine this with the next one?

Reply:
I think we should leave them separate so that they can link to their respective PRs & provide more detail in the latter one.

```suggestion
  - description: Security events are now included in the audit log by default to provide more visibility to security teams.
```